src/_pytest/python.py

```diff
@@ -1000,49 +1023,59 @@ def _resolve_arg_ids(
         :rtype: List[str]
         :return: the list of ids for each argname given
         """
-        idfn = None
```

Comment:
I rearranged the code here to make it amenable to mypy. I also think it's a bit clearer.